samples/IISSample/Startup.cs

```diff
@@ -15,13 +17,14 @@ public void ConfigureServices(IServiceCollection services)
             // These two middleware are registered via an IStartupFilter in UseIISIntegration but you can configure them here.
             services.Configure<IISOptions>(options =>
             {
+                options.AuthenticationDisplayName = "Windows Auth";
```

Comment:
Not sure about that. Do you want the Windows scheme to be included by default in the "providers list"?

Pretty sure this has already been discussed in the past.